Build failed: Link Iterable_React_Native_SDK (x86_64)

I'm getting this error when trying to build our iOS app:

```
Undefined symbols for architecture x86_64:
  "_OBJC_CLASS_$_RCTEventEmitter", referenced from:
      type metadata for Iterable_React_Native_SDK.ReactIterableAPI in ReactIterableAPI.o
  "_OBJC_METACLASS_$_RCTEventEmitter", referenced from:
      _OBJC_METACLASS_$_ReactIterableAPI in ReactIterableAPI.o
  "_RCTRegisterModule", referenced from:
      _initialize_ReactIterableAPI in RNIterableAPI.o
ld: symbol(s) not found for architecture x86_64
```

Details on my setup:
- Xcode 12.4 (12D4e)
- React Native 0.63.3
- Iterable-React-Native-SDK (1.1.0-beta1)
- Iterable-iOS-SDK (6.3.0-beta2)

I tried cleaning and removing derived data, without success so far. Any help appreciated!

--- a/Iterable-React-Native-SDK.podspec
+++ b/Iterable-React-Native-SDK.podspec
@@ -25,5 +25,5 @@ Pod::Spec.new do |s|
   s.swift_version = '5.3'
 
   s.dependency 'Iterable-iOS-SDK', '~> 6.3.0-beta2'
-  s.dependency 'React'
+  s.dependency 'React-Core'
 end

